src/components/StyleGuide/StyleGuide.js

```diff
@@ -0,0 +1,371 @@
+/* eslint-disable dot-notation */
+import React from 'react';
+import ColorRow from './ColorRow/ColorRow';
+import TypographyRow from './TypographyRow/TypographyRow';
+import style from './StyleGuide.scss';
+
+const COLORS = {
```

Comment:
OK there is a ridiculous amount of duplication between mixins, class names, and content in this file, but it's only for the style guide and should not change.